consul-lambda-registrator/trigger_test.go

```diff
@@ -10,13 +10,14 @@ import (
 
 	"github.com/hashicorp/consul/api"
 	"github.com/hashicorp/consul/sdk/testutil"
+	"github.com/hashicorp/terraform-aws-consul-lambda-registrator/consul-lambda-registrator/structs"
 )
 
 func TestAWSEventToEvents(t *testing.T) {
```

Comment:
I'm not seeing a test case here for skipping an event because the datacenter doesn't match. Maybe I missed it though.

Reply:
You didn't miss it, it wasn't there. I've added it. I've also created a ticket to add unit tests for the mTLS upsert and delete because those are missing. I will do that in a dedicated PR since this one is already way too big.